### PR TITLE
Fixes #1506 update value unstirred water

### LINKED ISSUE #1506: Value origin for unstirred water layer thickness is wrong
The value origin for the unstirred water layer thickness in the particle dissolution model seems to be wrong:
Annotation 2020-06-17 112529

In the [referenced publication](https://www.sciencedirect.com/science/article/pii/S0939641110001517), Table 3, a value of 30 µm is given.

@Incei 
@KatrinCoboeken

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Dump/PKSimDB_dump.txt [hunk 1]
```diff
@@ -3780,7 +3780,7 @@ INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_Subcutaneou
 INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_Tablet_alpha','ln(2)/DissTime50^DissShape','Inversed time');
 INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_Tablet_beta','0.92','Dimensionless');
 INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_w_fact','0.2','Dimensionless');
-INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_WaterLayerThickness','0.0002','Length');
+INSERT INTO tab_calculation_method_rates VALUES('Formulation','PARAM_WaterLayerThickness','0.0003','Length');
```
Comment (msevestre) [inline src/Db/Dump/PKSimDB_dump.txt:3783]: @Yuri05 Here we go. Let see if something breaks because of this in the tests

**Comment by Yuri05** [comment]
Yep, I think the value was wrong and 30 is indeed the correct value. It's also 30 in the initial paper (https://pubs.acs.org/doi/10.1021/la800694j) referenced by Willmann et al.

**Review by Yuri05 — APPROVED** [review]

**Comment by Yuri05** [comment]
That was clear...
grafik

We could change the thickness parameter in the reference simulation as well, then the tests should be green again
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/tests/PKSim.Tests/Data/PrototypeParticlesDissolution_722.pkml

**Comment by msevestre** [comment]
value changed. Test running again

**Comment by msevestre** [comment]
@Yuri05 I have no idea what this test is doing/ It is now failing for obscure reasons (variable became negative or outputs not the same). Can you check this out?

**Comment by Yuri05** [comment]
You changed the value in the building block in the pkml, but not in the simulation - in the simulation it is still 20.
If you change it in the simulation as well - should be fine (hopefully)
grafik

**Comment by msevestre** [comment]
need to update the DB> Merging and hoping for the best

File: tests/PKSim.Tests/IntegrationTests/SimulationWithParticlesFormulationSpecs.cs [hunk 2]
```diff
@@ -725,6 +725,9 @@ protected override void SetupSimulation()
       {
          //disable precipitation
          PrecipitatedDrugSoluble = true;
+
+         _simulation.Solver.AbsTol = 1e-10;
+         _simulation.Solver.RelTol = 1e-6;
```
Comment (msevestre) [inline tests/PKSim.Tests/IntegrationTests/SimulationWithParticlesFormulationSpecs.cs:730]: @Yuri05 This was required too.

**Pull request merged**